src/test/scala/latis/input/TestHapiJsonAdapter.scala

```diff
@@ -0,0 +1,47 @@
+package latis.input
+
+import java.net.URI
+
+import latis.data.{Data, Sample, SeqFunction}
+import latis.metadata.Metadata
+import latis.model._
+import latis.output.TextWriter
+import latis.util.FileUtils
+import org.scalatest.FlatSpec
+import org.scalatest.Matchers._
+
+class TestHapiJsonAdapter extends FlatSpec {
+  
+  val reader = new AdaptedDatasetReader {
+    //def uri: URI = new URI("https://cdaweb.gsfc.nasa.gov/hapi/data?id=AC_H0_MFI&time.min=2019-01-01&time.max=2019-01-02&parameters=Magnitude,dBrms&format=json")
+    def uri: URI = FileUtils.resolvePath("data/hapi_json_data.txt").get.toUri
+    def model: DataType = Function(
+      Scalar(Metadata("id" -> "Time", "type" -> "string")),
+      Tuple(
+        Scalar(Metadata("id" -> "Magnitude", "type" -> "float")),
+        Scalar(Metadata("id" -> "dBrms", "type" -> "float"))
+      )
+    )
+    def adapter = new HapiJsonAdapter(model)
+  }
+
+  val ds = reader.getDataset
+
+  "The first sample in the HAPI JSON dataset" should "contain the following values" in {
```

Comment:
I think the message after "should" should be changed to something more like "contain the correct values" because the test output will look something like this as is:

```
The first sample in the HAPI JSON dataset
  - should contain the following values
  - should ...
```

That looks strange to me, because from the test output there are no "following values."